sdk/nodejs/runtime/stack.ts

```diff
@@ -66,7 +67,7 @@ class Stack extends ComponentResource {
      *
      * @param init The callback to run in the context of this Pulumi stack
      */
-    private async runInit(init: () => Promise<Inputs>): Promise<Inputs | undefined> {
+    async initialize(args: { init: () => Promise<Inputs> }): Promise<Inputs> {
```

Comment:
stacks were always async constructed before.  Now, we can just use the built-in system for that.